.eslintrc.yml

```diff
@@ -8,6 +8,7 @@ plugins:
   - react-hooks
   - prettier
   - jest
+  - '@typescript-eslint'
```

Comment:
does `@` have special meaning in yaml that requires quoting?

Reply:
Yes

![image](https://github.com/user-attachments/assets/b97bc46e-9e5a-48c0-9aaf-e40be5ba0f7e)